packages/tocco-ui/src/Menu/ItemFlyout.js

```diff
@@ -0,0 +1,54 @@
+
+import React from 'react'
+import {theme} from 'styled-system'
+
+import Button from '../Button'
+import Item, {ItemStyles} from './Item'
+import {getElevation} from '../utilStyles'
+
+const ItemFlyoutStyles = ItemStyles.extend`
+  > ul {
+    position: absolute;
+    top: calc(100% + ${props => theme('space.2')});
+    left: 0;
+    z-index: 1;
+    background-color: ${props => theme('colors.base.paper')};
+
+    ${props => getElevation(props, 1)}
+
+    ul {
+      top: 0;
+      left: calc(100% + ${props => theme('space.2')});
+      right: auto;
+    }
+  }
+`
+class ItemFlyout extends Item {
```

Comment:
i prefer stateless react component in such cases. In my opinion are they more elegant and simpler to read.